service/matching/handler/interfaces.go

```diff
@@ -34,7 +34,8 @@ import (
 type (
 	// Engine exposes interfaces for clients to poll for activity and decision tasks.
 	Engine interface {
-		Stop()
+		common.Daemon
+
```

Comment:
the start call here doesn't do anything yet, but I was expecting that this will be sticky/stateful in a [future pr](https://github.com/uber/cadence/pull/6192)